src/controller/CHIPPersistentStorageDelegate.h

```diff
@@ -34,30 +67,34 @@ class DLL_EXPORT PersistentStorageDelegate
      *   Lookup the key and return it's stringified value
      *
      * @param[in] key Key to lookup
-     * @return Value or nullptr if not found. Lifetime of the returned
-     *         buffer is tied to the delegate object. If the delegate is
-     *         freed, the returned value would be inaccessible.
+     * @param[in] delegate Callback handler using which the value wiil be returned.
```

Comment:
I think the documentation for these methods should avoid the words "return ...", if a delegate method will be called instead